tests/contract_testcases/substrate/arrays/push_array_phi_node.sol

```diff
@@ -0,0 +1,16 @@
+// This tests that after a push/pop the correct phi node is created for array
+contract Array_bound_Test {
```

Comment:
What do you think of fixing the indentation of this file?

Reply:
Fixed